source/libs/cache.ts

```diff
@@ -1,26 +1,28 @@
-type CacheGetter = () => string|Promise<string>;
+ type CacheGetter<TValue extends any = any> = () => TValue | Promise<TValue>;
```

Comment:
This was only type as string because it was only used in the releases tab feature. Now it's generic even if it's only used in one place at the moment.